meilisearch-http/src/index/updates.rs

```diff
@@ -248,8 +254,21 @@ impl Index {
         }
 
         match settings.filterable_attributes {
-            Setting::Set(ref facet_types) => builder.set_filterable_fields(facet_types.clone()),
-            Setting::Reset => builder.set_filterable_fields(HashSet::new()),
+            Setting::Set(ref facets) => {
+                builder.set_filterable_fields(facets.clone().into_iter().collect())
+            }
+            Setting::Reset => builder.reset_filterable_fields(),
+            Setting::NotSet => (),
+        }
+
+        match settings.sortable_attributes {
+            Setting::Set(ref fields) => {
+                builder.set_sortable_fields(fields.into_iter().cloned().collect())
+            }
+            Setting::Reset => {
+                // TODO we must use the reset_sortable_fields in a futur PR.
```

Comment:
I forgot to add the `reset_sortable_fields` on the milli's side but it is no big deal as it is equivalent to set an empty list of fields. I create a PR on milli https://github.com/meilisearch/milli/pull/330.